distributed/scheduler.py

```diff
@@ -5575,7 +5575,7 @@ def handle_long_running(self, key=None, worker=None, compute_duration=None):
         duration accounting as if the task has stopped.
         """
         parent: SchedulerState = cast(SchedulerState, self)
-        if key not in parent._tasks:
+        if key not in parent._tasks:  # pragma: no cover
             logger.debug("Skipping long_running since key %s was already released", key)
```

Comment:
I'm not sure if the current behaviour is stable
```suggestion
        if key not in parent._tasks:
```